modules/admin/admin-ui/src/main/resources/web/admin/common/js/content/site/inputtype/siteconfigurator/SiteConfiguratorDialog.ts

```diff
@@ -74,6 +77,17 @@ module api.content.site.inputtype.siteconfigurator {
             });
         }
 
+        private handleDialogClose(formView: FormView) {
+            formView.getChildren().forEach((element: api.dom.Element) => {
+                if (api.ObjectHelper.iFrameSafeInstanceOf(element, InputView)) {
+                    const imageSelector = (<InputView> element).getInputTypeView().getElement();
```

Comment:
@mta I don't see any reason to use "const" instead of "var" or "let" here. Do you?